tests/system/marathon_common_tests.py

```diff
@@ -1237,3 +1240,184 @@ def test_ipv6_healthcheck(docker_ipv6_network_fixture):
         "The number of healthy tasks is {}, but {} was expected".format(app['tasksHealthy'], target_instances_count)
 
     client.remove_app(app['id'], True)
+
+
+@shakedown.dcos_1_11
+@pytest.mark.skipif("shakedown.ee_version() is None")
+def test_faultdomains_default():
+    """Tests if the applications that do not have a fault domain specified are launched on the same
+       fault domain as the master.
+    """
+
+    # Discover which is the "local" cluster domain
+    local_domain = get_cluster_local_domain()
+
+    # Get how many agents do we have in the local domain (zone and region) and try to start
+    # that many applications. Mesos should round-robbin the instances in the region
+    region_agents = count_agents_in_faultdomains(regions=local_domain.region)
+
+    # Create an app with default configuration
+    app_def = apps.faultdomain_app(instances=region_agents, suffix="-defaults")
```

Comment:
we prefer to not have constants as app ids... we should be consistent.   we should be able to launch multiple fault domain apps and not have collisions.   see other app launches as an example.

Reply:
This is not creating a constant app ID...
1. `apps.faultdomain_app` calls 
2. `apps.load_app` that uses
3. `utils.make_id` to create a unique ID
4. The `suffix` is appended to help debugging

Reply:
Please see @zen-dog's changes on app ids.

Reply:
I think we have resolved this. I am using `make_app` and passing the `suffix` as part of the app prefix 👍